web/client/components/import/dragZone/NormalContent.jsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2018, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const React = require('react');
+const { Glyphicon } = require('react-bootstrap');
+const DropText = require('./DropText');
+
+
+module.exports = (props) => (<div>
```

Comment:
I think we are splitting the code in too many pieces. I have a hard time trying to see the global picture.

Reply:
The global picture is in Content.jsx 
If you need detail of how the view is implemented, you go in the file. 
We can also reuse and generalize this pieces in the future.